src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -91,57 +129,17 @@
-    <Compile Include="System\Security\Policy\IMembershipCondition.cs" />
-    <Compile Include="System\Security\Policy\NetCodeGroup.cs" />
-    <Compile Include="System\Security\Policy\PermissionRequestEvidence.cs" />
+    <Compile Include="System\Security\PolicyLevelType.cs" />
+    <Compile Include="System\Security\SecurityContext.cs" />
+    <Compile Include="System\Security\SecurityContextSource.cs" />
     <Compile Include="System\Security\SecurityManager.cs" />
     <Compile Include="System\Security\SecurityState.cs" />
     <Compile Include="System\Security\SecurityZone.cs" />
     <Compile Include="System\Security\XmlSyntaxException.cs" />
-    <Compile Include="System\Security\PolicyLevelType.cs" />
-    <Compile Include="System\Security\SecurityContext.cs" />
-    <Compile Include="System\Security\SecurityContextSource.cs" />
-    <Compile Include="System\Security\PermissionSet.cs" />
-    <Compile Include="System\Security\NamedPermissionSet.cs" />
-    <Compile Include="System\Security\IEvidenceFactory.cs" />
-    <Compile Include="System\Security\HostSecurityManagerOptions.cs" />
-    <Compile Include="System\Security\HostSecurityManager.cs" />
-    <Compile Include="System\Security\HostProtectionException.cs" />
-    <Compile Include="System\Security\Permissions\FileIOPermissionAttribute.cs" />
-    <Compile Include="System\Security\Permissions\IsolatedStorageContainment.cs" />
-    <Compile Include="System\Security\Permissions\IsolatedStorageFilePermission.cs" />
-    <Compile Include="System\Security\Permissions\IsolatedStorageFilePermissionAttribute.cs" />
-    <Compile Include="System\Security\Permissions\IsolatedStoragePermission.cs" />
-    <Compile Include="System\Security\Permissions\IsolatedStoragePermissionAttribute.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermission.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermissionAccessEntry.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermissionAccessEntryCollection.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermissionAccessEntryEnumerator.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermissionAttribute.cs" />
-    <Compile Include="System\Security\Permissions\KeyContainerPermissionFlags.cs" />
-    <Compile Include="System\Security\Permissions\ResourcePermissionBase.cs" />
-    <Compile Include="System\Security\Permissions\ResourcePermissionBaseEntry.cs" />
-    <Compile Include="System\Security\Permissions\StorePermission.cs" />
-    <Compile Include="System\Security\Permissions\StorePermissionAttribute.cs" />
-    <Compile Include="System\Security\Permissions\StorePermissionFlags.cs" />
-    <Compile Include="System\Security\Permissions\TypeDescriptorPermissionAttribute.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
     <Reference Include="mscorlib" />
     <Reference Include="System" />
-  </ItemGroup>
-  <ItemGroup>
```

Comment:
My merging those two ItemGroups together you are condition the System.Security.AccessControl reference to only work on netfx